content/integrations/github.md

```diff
@@ -14,18 +14,41 @@ Capture GitHub commits in Datadog to:
   * Identify when new code changes lead to system alerts or build failures
   * Discuss code changes with your team in the Datadog Event Stream
 
-
+{{< img src="git_integration_screen.png" alt="Git event" >}}
 
 ## Configuration
 
+1. Add this URL to your Github Webhook, **Settings>Webhooks>Add new Webhook** with this URL:
+```
+https://app.datadoghq.com/intake/webhook/github?api_key=<YOUR_DATADOG_API_KEY>
+```
 
-Select 'Github' [on the account settings screen][1] and link your account. You can then select which repos you would like to integrate, which branches, and if you'd like to receive commits and/or issues.
+2. Select what you want to send to datadog, we support:
+     * Push (new commits), 
+     * Create and delete (for tags), 
+     * Pull requests,
+     * Issues, 
+     * All comments.
 
-## What to Expect
+3. Add the branches you wish to monitor for each repository. Wildcards are also supported to add all repositories for a user or Organization.
 
+## What to Expect
 
 Once the integration is complete, whatever you select (commits and/or issues) will populate into your Datadog Event Stream. If you view a dashboard, in the top left search bar you can type `sources:github` to see github events overlayed over your the graphs on that dashboard.
 
-   [1]: https://app.datadoghq.com/account/settings
+## FAQ
+
+**Why events don't appear to be showing up in my datadog event stream with my github integration?**
+
+If you have setup your webhook on the relevant Github repos and you can see it's sending data but events don't appear to be showing up in the event stream this might come from your webhook settings:
+
+Instead of having your webhook configured with `content-type:application/x-www-form-urlencoded`
```

Comment:
If your web hook is configured with `content-type:application/x-www-form-urlencoded`, set it to `content-type:application/json` instead.